misc/wiki/swagger/swagger.json

```diff
@@ -2,17 +2,18 @@
   "swagger": "2.0",
   "info": {
     "version": "2017-01-30T23:30:18Z",
-    "title": "{api_deployment_node_title}",
+    "title": "{apiDeploymentNodeTitle}",
```

Comment:
why this got changed? This should be same for core and user created service right? Are we using this in any place?

Reply:
We refer this file from https://github.com/tmobile/jazz/wiki/Supported-Service-Types-and-Code-Templates. These changes need to be reverted though.

Reply:
All the  changes except for adding `"operationId": "{operationId}"` should be reverted.

Reply:
fixed